infra/run_tests.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing perepo_managerissions and
+# limitations under the License.
+"""Runs the tests associated with the changes made to the repo."""
```

Comment:
Can this script be used to run all the tests? I think it's better to make it that way, and let the caller to specify a subset of tests to run (e.g. the presubmit script can do that for the affected files only, assuming there is a sound way to figure out all dependencies)

Reply:
I moved the contents of this script to presubmit.py. All tests can be run by running `sudo python3 presubmit.py tests`